noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr

```diff
@@ -7,8 +7,6 @@ use super::traits::{Deserialize, Packable, Serialize};
 /// - `typ`: The type of the struct or field being deserialized (e.g., a custom struct, array, or primitive).
 /// - `field_array_name`: The name of the field array containing serialized field data (e.g., `"values"`).
 /// - `num_already_consumed`: The number of fields already processed in previous recursion calls.
-/// - `to_replace`: The name of a specific field that should be replaced during deserialization.
-/// - `ro_replace_with`: The value to replace the `to_replace` field with (e.g., `NoteHeader::empty()`).
```

Comment:
These args were no longer needed as we no longer need to replace a header when deserializing.